web/backend/src/Server.ts

```diff
@@ -519,15 +536,23 @@ function sendToDela(dataStr: string, req: express.Request, res: express.Response
         .send(`failed to proxy request: ${req.method} ${uri} - ${error.message} - ${resp}`);
     });
 }
+app.put('/api/evoting/authorizations', (req) => {
```

Comment:
We use this endpoint to add authorization to manage a form for the user that created it. Then, only people that can create a form can "call" this endpoint. Therefore the check is done before when a user access the create form page. But maybe it can be a good idea to recheck the authorization of creating a form here.

Reply:
> Then, only people that can create a form can "call" this endpoint
This is not what the code is saying. Maybe the frontend doesn't that, but still anyone can call this endpoint and make himself an owner of an election, which is a sever security issue.

Reply:
Okk I see, I will than add a check of the authorization !
Thanks for the review !